fdbrpc/FailureMonitor.h

```diff
@@ -76,8 +76,8 @@ struct FailureStatus {
 	bool isFailed() const { return failed; }
 	bool isAvailable() const { return !failed; }
 
-	bool operator == (FailureStatus const& r) const { return failed == r.failed; }
-	bool operator != (FailureStatus const& r) const { return failed != r.failed; }
+	bool operator==(FailureStatus const& r) const { return failed == r.failed; }
```

Comment:
Are there functional changes to `FailureMonitor.actor.cpp` and  `FailureMonitor.h`? 
If not, it will be really helpful if you can have a separate PR for the formatting stuff, have that PR merged first and then merge master into this PR, so that this PR becomes much easier to digest.